### ISSUE TITLE
List additional dependencies

### ISSUE THREAD

**Issue by Flix6x** [SeitaBV/flexmeasures-entsoe#5]
Should we add a `requirements.in` file to state the dependency on `entsoe-py`?

**Comment by nhoening** [comment]
Yes. But that comes together with making this a real package I would say. I'm working on a new plugin template version where that stuff also comes built-in.

Am 13. September 2021 13:30:19 MESZ schrieb Felix Claessen ***@***.***>:
>Should we add a `requirements.in` file to state the dependency on
>`entsoe-py`?
>
>-- 
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/SeitaBV/flexmeasures-entsoe/issues/5

-- Gesendet von /e/ Mail.
